source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionForm.tsx

```diff
@@ -120,6 +126,30 @@ const AcquisitionDetailSubForm: React.FC<{
     }
   }, [fileStatusTypeCode, setFieldValue]);
 
+  React.useEffect(() => {
+    if (!ownerSolicitorContact || ownerSolicitorContact?.personId) {
+      setIsPersonSelected(true);
+    }
+
+    if (ownerSolicitorContact?.organizationId && !ownerSolicitorContact?.personId) {
+      setIsPersonSelected(false);
+    }
+  }, [ownerSolicitorContact]);
+
+  const { organization } = useOrganizationDetail(ownerSolicitorContact?.organizationId ?? 1);
```

Comment:
this seems undesireable, why make a request with id 1 when no organization is set? better to only make requests when the id is set.

Reply:
react doesnt allow conditional usage of hooks so one way to do that would be to wrap the call to useOrganizationDetails in wrapper component and communicate the organization info back to the parent component via state.